boardClass.lua

```diff
@@ -4,6 +4,7 @@ import "CoreLibs/sprites"
 import "CoreLibs/timer"
 
 local gfx <const> = playdate.graphics
+local possibleNumberFont = gfx.font.new("font-Bitmore")
```

Comment:
so we only read the font once, and then reuse it. this particular update fixed the lag on my device